scripts/seek/tests/test_seekCreateDB.py

```diff
@@ -22,7 +22,7 @@ def setup_class(cls):
             cls.temp_dir = tempfile.TemporaryDirectory()
             tmpDirName = cls.temp_dir.name
         else:
-            tmpDirName = '/tmp/testSeekUtils'
+            tmpDirName = '/tmp/testSeekCreateDb'
```

Comment:
Funny, because locally I changed my temp dir to this path as well to distinguish it from other tests.